frappe/utils/pdf.py

```diff
@@ -352,6 +353,18 @@ def toggle_visible_pdf(soup):
 		tag.extract()
 
 
+@frappe.whitelist()
+@redis_cache(ttl=60 * 60)
+def is_wkhtmltopdf_valid():
+	try:
+		res = subprocess.check_output(["wkhtmltopdf", "--version"])
+		is_wkhtmltopdf_valid = "qt" in res.decode("utf-8").lower()
+	except Exception:
+		pass
+
+	return is_wkhtmltopdf_valid
```

Comment:
This will break in the case of an exception in the subprocess call, since the variable is defined after.
Maybe just `return False` for exceptions, or declare the variable outside the try block with a false value.